### PR TITLE
Add Balazs's features to collection

### PR DESCRIPTION
**Pull request by Balazs23**
I've made two features (bazel, nx) what I was missing from the feature list

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -243,3 +243,8 @@
   contact: https://github.com/ChristopherMacGown/devcontainer-features/issues
   repository: https://github.com/ChristopherMacGown/devcontainer-features
   ociReference: ghcr.io/christophermacgown/devcontainer-features
+- name: Devcontainer features by @Balazs
+  maintainer: Balazs23
+  contact: https://github.com/Balazs23/devcontainers-features/issues
+  repository: https://github.com/Balazs23/devcontainers-features
+  ociReference: ghcr.io/balazs23/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/balazs23/devcontainers-features
```
Can we update to match the `ociReference` here? Thanks! 

image

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Nice, left one comment!

File: _data/collection-index.yml (hunk 1 above)
Comment (Balazs23) [reply _data/collection-index.yml]: Fixed, thank you!

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Looks like there is a merge conflict, can you help fix that? thanks!

**Comment by Balazs23** [comment]
> Looks like there is a merge conflict, can you help fix that? thanks!

Done :)

**Review by samruddhikhandale — APPROVED** [review]
Thank you! 🚀

**Pull request merged**